index.html

```diff
@@ -0,0 +1,121 @@
+            <input
+                class="stretched"
+                type="number"
+                id="number"
+                placeholder="Age"
+                min="13"
+                max="200"
+            /><br />
+            <label for="">Which option best describe your current role?</label
+            ><br />
+
+            <select name="select option" id="dropdown" class="stretched">
+                <option value="Student">Student</option>
+                <option value="Full Time job">Full Time job</option>
+                <option value="Full timer learner">Full timer learner</option>
+                <option value="Prefer not to say">Prefer not to say</option>
+                <option value="others">others</option>
+            </select>
+
+            <div id="radio-buttons">
+                <p>Would you recommend us?</p>
+                <input
+                    type="radio"
+                    name="recommend"
+                    value="yes"
+                    id="radio-btn"
+                />
+                <label for="radio-btn">Yes</label>
+                <input
+                    type="radio"
+                    name="recommend"
+                    value="no"
+                    id="radio-btn"
+                />
+                <label for="radio-btn">No</label>
+            </div>
+
+            <div id="checkboxes">
+                <p>What would you like to see improved?</p>
+                <input
+                    type="checkbox"
+                    value="frontend"
+                    name="improvements"
+                    id="checkbox"
+                />
+                <label for="checkbox">Frontend</label>
+                <input
+                    type="checkbox"
+                    value="database"
+                    idname="improvements"
+                    id="checkbox"
+                />
+                <label for="checkbox">Database</label>
+                <input
+                    type="checkbox"
+                    value="backend"
+                    name="improvements"
+                    id="checkbox"
+                />
+                <label for="checkbox">Backend</label>
+            </div>
+
+            <textarea
+                name="comments"
+                id="texarea"
+                cols="30"
+                rows="10"
+            ></textarea>
+            <br />
+            <button type="submit" id="submit">Submit</button>
+        </form>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+        <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+        <script src="js/index.js"></script>
+    </body>
+</html>
```

Comment:
Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)